### DISCUSSION TITLE
Kong Plugin certification

### DISCUSSION THREAD

**Discussion by joshi-mohit** [Kong/kong#10418] [Help] [answered]
Team, we at AppSentinels have build an API security Platform. This platform integrates with API-GW to provide frcitionless integration with customer network.
On similar lines we have a plugin for Kong and some of our customers are asking for certified Kong Plugin. The kobg website does provides plugins in the 'Kong Plugin Hub'.
Wanted to know understamd on how this certification works and how we can make our Plugin available at "Kong Plugin Hub" so a certified version is available to Kong customers also.
Need help and guidance here

**Comment by ahuffman** [comment, marked as answer]
Hi @joshi-mohit, we're currently refining the process on how we publish 3rd party plug-ins to the plugin-hub.  We will have more details in the next couple months.  

Until then, we are not taking any submissions outside of partner agreements and commitments we may have already had.  We will fill you in on the new process once that opens.  Thank you for your patience.

**Reply by joshi-mohit** [reply]
Thanks Andrew. Kong customers are pushing for this certification before production deployments. Please do let me know any details as and when are available on same. Thanks Mohit

**Comment by joshi-mohit** [comment]
Hi Andrew
   Hope this message finds you at your best.
Picking up the threads from your last mail, wanted to check with you if this process change is complete and how we can proceed on this.

Please do let me know any good when we get into a discussion to take this forward.

Thanx
Mohit

**Comment by ahuffman** [comment]
Hi @joshi-mohit , sorry for the delayed response to your question.

Currently, the Kong Plugin Hub is only open for publishing by Kong Partners.

To initiate the process, you must first fill out a form to sign up to [become a Kong partner](https://konghq.com/partners/become-a-partner).

Once that is complete we can schedule a session to run through the rest of the partnering requirements.

We require a few things once you've signed up to become a partner.

1. 10 minute or less video recording demonstrating the integration you've built
2. 1 page document outlining how the plugin works, architecture diagram, and any other requirements to enable the integration
3. After we've received items 1 & 2 we will schedule an additional session to run through any technical questions our Partner Engineering team may have, and collect your plugin code for review
4. After plugin code review, we can then have you initiate a pull request to submit your Kong Plugin Hub plugin documentation, which will require a final review prior to publishing.

We look forward to your submission, and to working with you and your team!

Thank you,
Andrew
